functions/go/set-labels/transformer/labels.go

```diff
@@ -0,0 +1,235 @@
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "Deployment"):
+			path = "spec/template/spec/affinity/podAffinity/requiredDuringSchedulingIgnoredDuringExecution/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "Deployment"):
+			path = "spec/template/spec/affinity/podAntiAffinity/preferredDuringSchedulingIgnoredDuringExecution/podAffinityTerm/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "Deployment"):
+			path = "spec/template/spec/affinity/podAntiAffinity/requiredDuringSchedulingIgnoredDuringExecution/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "Deployment"):
+			path = "spec/template/spec/topologySpreadConstraints/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("", "", "ReplicaSet"):
+			path = "spec/selector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, true)
+		case o.IsGVK("", "", "ReplicaSet"):
+			path = "spec/template/metadata/labels"
+			err = updateLabels(o, path, p.NewLabels, true)
+		case o.IsGVK("", "", "DaemonSet"):
+			path = "spec/selector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, true)
+		case o.IsGVK("", "", "DaemonSet"):
+			path = "spec/template/metadata/labels"
+			err = updateLabels(o, path, p.NewLabels, true)
+		case o.IsGVK("apps", "", "StatefulSet"):
+			path = "spec/selector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, true)
+		case o.IsGVK("apps", "", "StatefulSet"):
+			path = "spec/template/metadata/labels"
+			err = updateLabels(o, path, p.NewLabels, true)
+		case o.IsGVK("apps", "", "StatefulSet"):
+			path = "spec/template/spec/affinity/podAffinity/preferredDuringSchedulingIgnoredDuringExecution/podAffinityTerm/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "StatefulSet"):
+			path = "spec/template/spec/affinity/podAffinity/requiredDuringSchedulingIgnoredDuringExecution/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "StatefulSet"):
+			path = "spec/template/spec/affinity/podAntiAffinity/preferredDuringSchedulingIgnoredDuringExecution/podAffinityTerm/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "StatefulSet"):
+			path = "spec/template/spec/affinity/podAntiAffinity/requiredDuringSchedulingIgnoredDuringExecution/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "StatefulSet"):
+			path = "spec/template/spec/topologySpreadConstraints/labelSelector/matchLabels"
+			err = updateLabels(o, path, p.NewLabels, false)
+		case o.IsGVK("apps", "", "StatefulSet"):
+			path = "spec/volumeClaimTemplates[]/metadata/labels"
```

Comment:
volumeClaimTemplates is a slice. (Suggest checking the struct of `StatefulSet`)? line 213 won't be able to handle this type. please consider using the SDK method.